<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_dialog</class>
 <widget class="QDialog" name="settings_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(37, 37, 38);
}

QLabel, QCheckBox {
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::RestoreDefaults</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="option_layout">
    <item>
     <widget class="QCheckBox" name="lp_mdls_cb">
      <property name="text">
       <string>Low Poly Models</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mtls_cb">
      <property name="text">
       <string>Use Materials</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bloom_cb">
      <property name="text">
       <string>Use Bloom</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="checkBox_layout">
      <item row="0" column="0">
       <widget class="QLabel" name="ao_label">
        <property name="text">
         <string>Ambient Occlusion Quality:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="ssao_cb">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="msaa_cb">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32x</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="msaa_label">
        <property name="text">
         <string>Multisample Level:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fxaa_cb">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extreme</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="lang_cb">
        <item>
         <property name="text">
          <string>Automatic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Portuguese (PT)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Portuguese (BR)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">German</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">French</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Italian</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lang_label">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fxaa_label">
        <property name="text">
         <string>FXAA Level:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="res_label">
        <property name="text">
         <string>Resolution:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="res_cb"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
